MomentumDiscordBot/Commands/Moderator/ModeratorCustomModule.cs

```diff
@@ -0,0 +1,146 @@
+                await Config.SaveToFileAsync();
+                await ReplyNewEmbedAsync(context, $"Command '{name}' added", MomentumColor.Blue);
+            }
+            else
+                await ReplyNewEmbedAsync(context, "Failed to add command", MomentumColor.Red);
+        }
+
+        [SlashCommand("remove", "deletes a custom commands")]
+        public async Task RemoveCustomCommandAsync(InteractionContext context, [Autocomplete(typeof(AutoCompleteProvider))][Option("name", "name of the custom command")] string name)
+        {
+            if (Config.CustomCommands.TryRemove(name, out _))
+            {
+                await Config.SaveToFileAsync();
+                await ReplyNewEmbedAsync(context, $"Command '{name}' removed", MomentumColor.Blue);
+            }
+            else
+            {
+                await ReplyNewEmbedAsync(context, "Failed to remove command.", MomentumColor.Blue);
+            }
+        }
+
+        [SlashCommand("rename", "deletes a custom commands")]
+        public async Task RenameCustomCommandAsync(InteractionContext context, [Autocomplete(typeof(AutoCompleteProvider))][Option("oldName", "name of the custom command")] string oldName, [Option("newName", "the new name")] string newName)
+        {
+            if (Config.CustomCommands.ContainsKey(newName))
+                await ReplyNewEmbedAsync(context, "Command '" + newName + "' already exists!", MomentumColor.Red);
+
+            else if (Config.CustomCommands.ContainsKey(oldName))
+            {
+                CustomCommand command;
+                if (!Config.CustomCommands.TryGetValue(oldName, out command))
+                    await ReplyNewEmbedAsync(context, "Failed to get old command value.", MomentumColor.Red);
+                else if (!Config.CustomCommands.TryAdd(newName, command))
+                    await ReplyNewEmbedAsync(context, "Failed to add new command.", MomentumColor.Red);
+                else if (!Config.CustomCommands.TryRemove(oldName, out _))
+                    await ReplyNewEmbedAsync(context, "Failed to remove old command.", MomentumColor.Red);
+                else
+                {
+                    await Config.SaveToFileAsync();
+                    await ReplyNewEmbedAsync(context, $"Command '{oldName}' was renamed to '{newName}'", MomentumColor.Blue);
+                }
+            }
+            else
+            {
+                await ReplyNewEmbedAsync(context, $"Command '{oldName}' doesn't exist", MomentumColor.Red);
+            }
+        }
+
+        [SlashCommand("list", "lists all custom commands")]
+        public async Task ListCustomCommandAsync(InteractionContext context, [Option("page", "if there are more than 25 command you can specify which part you want")] long page = 1)
+        {
+            string title = "Info Commands";
+            const int itemsPerPage = 25;
+            int pages = (int)Math.Ceiling((double)Config.CustomCommands.Count() / itemsPerPage);
```

Comment:
You can actually use the `.Count` *property* here rather than the `.Count()` *method*. (I'm cheating - Rider analysis pointed this out)